renderer/backend/metal/render_pass_mtl.mm

```diff
@@ -505,6 +505,12 @@ static bool Bind(PassBindingsCache& pass,
     }
   }
 
+  if (command.index_count == 0u) {
+    // Essentially a no-op. Don't record the command but this is not necessary
+    // an error either.
+    return true;
+  }
```

Comment:
Nit: We also do this check in `RenderPassMTL::EncodeCommands`. Should we remove it from there since this happens way earlier in the pipe?

Reply:
I'd prefer to leave that in there as a sort of paranoid check. I'll remove the validation log.